tests/sentry/api/endpoints/test_user_notification_settings.py

```diff
@@ -0,0 +1,85 @@
+from sentry.models import NotificationSetting
+from sentry.notifications.types import NotificationSettingOptionValues, NotificationSettingTypes
+from sentry.testutils import APITestCase
+from sentry.types.integrations import ExternalProviders
+
+FEATURE_NAMES = [
+    "organizations:notification-platform",
+]
+
+
+class UserNotificationSettingsTestBase(APITestCase):
+    endpoint = "sentry-api-0-user-notification-settings"
+
+    def setUp(self):
+        self.login_as(self.user)
+        self.org = self.organization  # Force creation.
```

Comment:
what happens if you don't do this?

Reply:
`self.organization` is a lazy attribute:
```
@fixture
def organization(self):
  return self.create_organization(name="foo", owner=self.user)
```
That means that the organization is only created (added to the DB) when I call `.organization()` on `self`.